modules/oe_content_departments_field/config/install/field.storage.node.oe_departments.yml

```diff
@@ -13,7 +13,7 @@ settings:
 module: rdf_skos
 locked: false
 cardinality: -1
-translatable: true
+translatable: false
```

Comment:
Let's use `true` here as we did for the other `skos_concept_entity_reference` fields.